docs/contributing/Compiler Test Plan.md

```diff
@@ -30,7 +30,8 @@ This document provides guidance for thinking about language interactions and tes
 - Performance and stress testing
  
 # Type and members
-- Access modifiers (public, protected, internal, protected internal, private protected, private), static modifier
+- Access modifiers (public, protected, internal, protected internal, private protected, private), static, ref
+    - If adding a new modifier, check modifiers for accessors as well.
```

Comment:
Unclear what this means at a casual reading.

Reply:
Will modify.

---
In reply to: [159701433](https://github.com/dotnet/roslyn/pull/23905#discussion_r159701433) [](ancestors = 159701433)